lib/pbench/test/unit/server/test_user_management_cli.py

```diff
@@ -0,0 +1,284 @@
+    ):
+        with client:
+            runner = CliRunner(mix_stderr=False)
+            result = runner.invoke(
+                user_create,
+                args=[
+                    TestUserManagement.USER_SWITCH,
+                    TestUserManagement.USER_TEXT,
+                    TestUserManagement.EMAIL_SWITCH,
+                    TestUserManagement.EMAIL_TEXT,
+                    TestUserManagement.FIRST_NAME_SWITCH,
+                    TestUserManagement.FIRST_NAME_TEXT,
+                    TestUserManagement.LAST_NAME_SWITCH,
+                    TestUserManagement.LAST_NAME_TEXT,
+                ],
+                input=f"{TestUserManagement.PSWD_TEXT}\n",
+            )
+            assert result.exit_code == 0, result.stderr
+            assert (
+                result.stdout
+                == f"{TestUserManagement.PSWD_PROMPT}\n" + "User test_user registered\n"
+            )
+
+    @staticmethod
+    @responses.activate
+    def test_admin_user_creation(client, server_config, pytestconfig):
+        with client:
+            runner = CliRunner(mix_stderr=False)
+            result = runner.invoke(
+                user_create,
+                args=[
+                    TestUserManagement.USER_SWITCH,
+                    TestUserManagement.USER_TEXT,
+                    TestUserManagement.PSWD_SWITCH,
+                    TestUserManagement.PSWD_TEXT,
+                    TestUserManagement.EMAIL_SWITCH,
+                    TestUserManagement.EMAIL_TEXT,
+                    TestUserManagement.FIRST_NAME_SWITCH,
+                    TestUserManagement.FIRST_NAME_TEXT,
+                    TestUserManagement.LAST_NAME_SWITCH,
+                    TestUserManagement.LAST_NAME_TEXT,
+                    TestUserManagement.ROLE_SWITCH,
+                    "ADMIN",
+                ],
+            )
+            assert result.exit_code == 0, result.stderr
+            assert result.stdout == "Admin user test_user registered\n"
+
+    @staticmethod
+    @responses.activate
+    def test_user_creation_with_invalid_role(client, server_config, pytestconfig):
+        with client:
+            runner = CliRunner(mix_stderr=False)
+            result = runner.invoke(
+                user_create,
+                args=[
+                    TestUserManagement.USER_SWITCH,
+                    TestUserManagement.USER_TEXT,
+                    TestUserManagement.PSWD_SWITCH,
+                    TestUserManagement.PSWD_TEXT,
+                    TestUserManagement.EMAIL_SWITCH,
+                    TestUserManagement.EMAIL_TEXT,
+                    TestUserManagement.FIRST_NAME_SWITCH,
+                    TestUserManagement.FIRST_NAME_TEXT,
+                    TestUserManagement.LAST_NAME_SWITCH,
+                    TestUserManagement.LAST_NAME_TEXT,
+                    TestUserManagement.ROLE_SWITCH,
+                    "ADMN",
+                ],
+            )
+            assert result.exit_code == 2
```

Comment:
It might be helpful to dump `result.stderr` here (e.g., see line 109, et al.).  If we get the _expected_ error, then nothing will be displayed, but if we get an exit status of `1`, it would be nice to know why.